components/post-image/PostImage.tsx

```diff
@@ -0,0 +1,47 @@
+'use client';
+import React from 'react';
+import Image from 'next/image';
+import { EIConTypes, Icon } from '@ost-cas-fee-adv-23-24/elbmum-design';
+
+interface IProps {
+  src: string;
+  alt: string;
+}
+
+const PostImage = ({ src, alt }: IProps) => {
+  return (
+    <div
+      onClick={() => {
+        alert('clicked');
+      }}
+      className="c-mumble-image rounded-2xl relative h-0 overflow-hidden group cursor-pointer"
+      style={{ paddingBottom: `${(8 / 17) * 100}%` }} // aspect ratio 17/8
```

Comment:
instead of using a the style prop, you could use the calc() function with tailwind:
pb-[calc((8/17)*100%]"

Reply:
done